### DISCUSSION TITLE
beatmap card audio preview in single thread mode can crash

### DISCUSSION THREAD

**Discussion by mouzedrift** [ppy/osu#15816] [Q&A] [answered]
when running the game in single thread mode and you preview multiple audios in quick succession the game can crash.

possibly the same or similar issue to #14313?

https://user-images.githubusercontent.com/43358824/143555792-6f840cfc-762a-4493-a53b-ffad17c6c9b7.mp4

version: c7f905f8dac24d19fb9e2b190ca6ea1600daa4a2

logs:
[performance.log](https://github.com/ppy/osu/files/7607527/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/7607528/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/7607530/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/7607531/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/7607532/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/7607533/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7607534/updater.log)
[database.log](https://github.com/ppy/osu/files/7607535/database.log)
[network.log](https://github.com/ppy/osu/files/7607536/network.log)

**Comment by bdach** [comment, marked as answer]
Resolved via #15821
